script/install/script.rb

```diff
@@ -44,26 +72,58 @@ def copy_theme_files
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/datatables', 'javascripts')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/jquery', 'javascripts')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/jquery-easing', 'javascripts')
-  cp_start_bootstrap_dir_to_target_assets_dir('vendor/fontawesome-free/css', 'stylesheets')
   cp_start_bootstrap_dir_to_target_assets_dir('vendor/fontawesome-free/webfonts', 'images')
+
+  cp_fontawesome_css_files
+
+  delete_map_files
 end
 
-def update_paths_in_theme_files
-  Pathname.new(TARGET_VIEWS_PATH).glob('*.html').each do |pathname|
-    contents = pathname.read
+def update_paths_in_html_files
+  Pathname.new(TARGET_VIEWS_PATH).glob('*.html.erb').each do |pathname|
+    updated_contents = pathname.read
+
+    updated_contents.gsub!(%r{<link href="vendor/(.*?)".*\>}) do
+      "<%= stylesheet_link_tag 'start_bootstrap_admin_theme/vendor/#{$1}' %>"
+    end
+
+    updated_contents.gsub!(%r{<link href="css/(.*?)".*\>}) do
+      "<%= stylesheet_link_tag 'start_bootstrap_admin_theme/css/#{$1}' %>"
+    end
+
+    updated_contents.gsub!(%r{<script src="vendor/(.*?)".*\>}) do
+      "<%= javascript_include_tag 'start_bootstrap_admin_theme/vendor/#{$1}' %>"
+    end
+
+    updated_contents.gsub!(%r{<script src="js/(.*?)".*\>}) do
+      "<%= javascript_include_tag 'start_bootstrap_admin_theme/js/#{$1}' %>"
+    end
+
+    updated_contents.gsub!(%r{src="img/(.*?)"}) do
+      "src='<%= image_path('start_bootstrap_admin_theme/img/#{$1}')%>'"
+    end
+
+    pathname.write(updated_contents)
+  end
+end
+
+def update_paths_in_fontawesome_css_files
+  fontawesome_stylesheets_target_path.glob('*.css.erb').each do |pathname|
+    updated_contents = pathname.read
+
+    updated_contents.gsub!(%r{url\("\.\.(.*?)"\)}) do
+      %Q{url("<%= asset_path('start_bootstrap_admin_theme/vendor/fontawesome-free/#{$1}') %>")}
+    end
 
-    updated_contents =
-      contents
-        .gsub('href="vendor/', 'href="/assets/start_bootstrap_admin_theme/vendor/')
-        .gsub('href="css/', 'href="/assets/start_bootstrap_admin_theme/css/')
-        .gsub('src="vendor/', 'src="/assets/start_bootstrap_admin_theme/vendor/')
-        .gsub('src="js/', 'src="/assets/start_bootstrap_admin_theme/js/')
-        .gsub('src="img/', 'src="/assets/start_bootstrap_admin_theme/img/')
+    updated_contents.gsub!(%r{url\(\.\.(.*?)\)}) do
+      %Q{url(<%= asset_path('start_bootstrap_admin_theme/vendor/fontawesome-free/#{$1}') %>)}
+    end
```

Comment:
It might be better to include Font Awesome's SCSS files instead of changing the paths in the CSS files. Can be done in the future.